### ISSUE TITLE
Couple things I noticed different from instructions - RPi2 + TF r1.3

### ISSUE THREAD

**Issue by EMCP** [samjabrahams/tensorflow-on-raspberry-pi#117]
### Describe the Issue
Walking through the step by step instructions, on a RPi2 , Ubuntu Mate 16.x, and TF (r1.3) source

### Steps to Reproduce

Download TF source and switch into release branch `r1.3`. follow instructions from there.

### Hardware/Software Info

Please provide the following information about your Raspberry Pi setup:

* Raspberry Pi model: Rpi 2 B+
* Operating System used: Ubuntu Mate 16.04
* Version of Python used: Python 3.5
* SD card memory size: 16GB
* Size of USB/other device used as swap (if building from source): Sandisk Swap disk 16GB
* TensorFlow git commit hash (if building from source): 

### Relevant Console Output/Logs

on sed command I saw a few warnings, nothing seemed majorly wrong though

```
pi@pi-desktop:~/Dev/git/tensorflow/tensorflow$ grep -R1 'lib64' | xargs sed -i 's/lib64/lib/g'
xargs: unmatched double quote; by default quotes are special to xargs unless you use the -0 option
sed: can't read CONTRIBUTING.md-: No such file or directory
sed: can't read ```bash: No such file or directory
sed: can't read CONTRIBUTING.md:: No such file or directory
sed: can't read export: No such file or directory
sed: can't read LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH: No such file or directory
sed: can't read CONTRIBUTING.md-: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD-: No such file or directory
sed: can't read cuda/nvvm/libdevice/libdevice.compute_35.10.bc,: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD:: No such file or directory
sed: can't read cuda/nvvm/lib64/libnvvm.so.3,: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD:: No such file or directory
sed: can't read cuda/nvvm/lib64/libnvvm.so,: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD:: No such file or directory
sed: can't read cuda/nvvm/lib64/libnvvm.so.3.1.0,: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD-: No such file or directory
sed: can't read cuda/nvvm/include/nvvm.h,: No such file or directory
sed: can't read --: No such file or directory
sed: can't read third_party/toolchains/gpus/cuda/BUILD-: No such file or directory
sed: can't read cmd: No such file or directory
sed: can't read =: No such file or directory
```

On the `WORKSPACE` editing part, my `WORKSPACE` file is much smaller than what is cited 
```
"Around line 283, change https to http"
...
```

I don't really see any lines at all regarding this package.. just going to ignore this instruction but thought I'd let you know.

**Comment by EMCP** [comment]
additionally the build failed, I am guessing by your findings you would suggest to just retry?

```
ERROR: /home/pi/Dev/git/tensorflow/tensorflow/tensorflow/core/kernels/BUILD:2183:1: C++ compilation of rule '//tensorflow/core/kernels:svd_op' failed: gcc failed: error executing command 
  (cd /home/pi/.cache/bazel/_bazel_altranpi/ffb4c04ea5cb888c1afd95fb0e50dd0b/execroot/tensorflow && \
  exec env - \
    PATH=/home/pi/bin:/home/altranpi/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin \
    PYTHON_BIN_PATH=/usr/bin/python3 \
    PYTHON_LIB_PATH=/usr/local/lib/python3.5/dist-packages \
    TF_NEED_CUDA=0 \
    TF_NEED_OPENCL=0 \

```

**Comment by turboencabulators** [comment]
I tried this today on my RPi 3B running Raspbian, 32GB SD card and 2 GB Swap memory and failed as well. Got Bazel to build with v0.6.0, which was required to build TF v1.3. 
When trying compiling/building TF, it failed with error `Server terminated abruptly (error code 1, error message" 'Received RST_STREAM err=8'`. Looks like I need a bigger swap USB memory stick!

Notes:
1. This is obvious, but with the current version of Bazel, I also noticed that the file `tools/cpp/cc_configure.bzl` is not where you include the command `return arm`. Instead it goes into a file pointed by it (don't recall the file name right now unfortunately).
2. Finally, like you,  I did not edit the Bazel `WORKSPACE` file because it is much smaller and does not have a line 283.
